### PR TITLE
feat: support clean read base on fast mode

### PR DESCRIPTION
**Pull request by hongyunyan**
### What problem does this PR solve?

Issue Number: ref #5252 

Problem Summary:

### What is changed and how it works?

enable clean read(optimization handle column) on fast mode.

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

image

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

**Comment by hongyunyan** [comment]
@lidezhu @JaySon-Huang

File: dbms/src/Storages/DeltaMerge/tests/gtest_dm_delta_merge_store_for_fast_mode.cpp [hunk 1]
```diff
@@ -1455,6 +1455,116 @@ try
...
+                                             {RowKeyRange::newAll(store->isCommonHandle(), store->getRowKeyColumnSize())},
+                                             /* num_streams= */ 1,
+                                             /* max_version= */ std::numeric_limits<UInt64>::max(),
+                                             EMPTY_FILTER,
+                                             TRACING_NAME,
+                                             /* is_raw_read= */ true,
+                                             /* expected_block_size= */ 1024)[0];
+        size_t num_rows_read = 0;
+
+        while (Block block = in->read())
```
Comment (JaySon-Huang) [inline dbms/src/Storages/DeltaMerge/tests/gtest_dm_delta_merge_store_for_fast_mode.cpp:1559]: Better to call `in->readPrefix` and `in->readSuffix` for the stream

**Review by JaySon-Huang — COMMENTED** [review]
Bascially LGTM

File: dbms/src/Storages/DeltaMerge/tests/gtest_dm_delta_merge_store_for_fast_mode.cpp [hunk 2]
```diff
@@ -290,7 +290,7 @@ try
                                              /* is_raw_read= */ true,
                                              /* expected_block_size= */ 1024)[0];
         size_t num_rows_read = 0;
-
+        in->readSuffix();
```
Comment (JaySon-Huang) [inline dbms/src/Storages/DeltaMerge/tests/gtest_dm_delta_merge_store_for_fast_mode.cpp:293]:
```suggestion
        in->readPrefix();
```

**Review by JaySon-Huang — APPROVED** [review]
LGTM

**Pull request merged**